### PR TITLE
[DC-110] Updated style on selected filters to have higher contrast

### PR DESCRIPTION
**Pull request by lu-c**
Before
Screen Shot 2022-01-12 at 1 43 47 PM

After
Screen Shot 2022-01-12 at 1 43 37 PM

Increased contrast, bolded text on select

### CODE REVIEW

**Review by petesantos — APPROVED** [review]
LGTM - I am assuming the test diff will go away once 2763 is merged

**Comment by lu-c** [comment]
> LGTM - I am assuming the test diff will go away once 2763 is merged

Oh shoot, i didnt notice i didnt create a new branch for this >.< Thanks! Yeah, sorry about that!

File: integration-tests/tests/run-catalog-filter.js
Comment (samanehsan) [inline integration-tests/tests/run-catalog-filter.js:60]: Should this throw an error if `datasetSizeAfterClear !== totalDatasetSize` instead?

Comment (lu-c) [reply integration-tests/tests/run-catalog-filter.js:60]: The text filter is still applied at this point of the unit tests, so this test matches the behavior of the application. I guess the question we should ask is whether or not the clear button should also clear out the search text too?

Comment (samanehsan) [reply integration-tests/tests/run-catalog-filter.js:60]: Oh I see! Then in the error message, should it include the `datasetSizeAfterSearch` instead of the `totalDatasetSize`?

Comment (samanehsan) [reply integration-tests/tests/run-catalog-filter.js:60]: > I guess the question we should ask is whether or not the clear button should also clear out the search text too?

Good point! Maybe @lstrano-broad has some insight?

Comment (lu-c) [reply integration-tests/tests/run-catalog-filter.js:60]: Ah, good point, I'll update the error message!

**Review by fboulnois — APPROVED** [review]

**Pull request merged**
